__tests__/app.test.js

```diff
@@ -9,6 +9,36 @@ beforeEach(() => seed(data))
 afterAll(() => db.end())
 
 describe('NC News Server', () => {
+    describe.only('GET 404 /invalid path', () => {
```

Comment:
Don't leave .only in your test file. Remove this and resubmit. I can't be sure the rest of your tests are passing and cannot review them at this time.